x/programs/rust/expose_macro/src/lib.rs

```diff
@@ -4,8 +4,8 @@ use core::panic;
 
 use proc_macro::TokenStream;
 use quote::{quote, ToTokens};
+use syn::__private::Span;
```

Comment:
You definitely should not be using `__private` haha. They probably leak what should be private types which is why that module exists and actually isn't private. [The example](https://docs.rs/syn/latest/syn/struct.Ident.html#examples) shows that you should be able to use `use proc_macro2::{Ident, Span};`.

Reply:
(required change)